README.md

```diff
@@ -280,20 +280,53 @@ smtpmock.ConfigurationAttr{
 package main
 
 import (
+  "bytes"
   "fmt"
   "net"
   "net/smtp"
 
   smtpmock "github.com/mocktools/go-smtp-mock/v2"
 )
 
+// User-defined loggers can be defined by implementing the Logger interface.
```

Comment:
@Hsn723 Maybe is a good point to add additional subsection for https://github.com/mocktools/go-smtp-mock?tab=readme-ov-file#manipulation-with-server, "Using custom logger" or something else with full example of this? In order not to confuse users with minimum library configuration. What do you think?

Reply:
@bestwebua Sure thing. I've reworked the documentation to move the custom logger to its own subsection: https://github.com/mocktools/go-smtp-mock/pull/201/commits/93075fac71b2d05e164a5f65a11a5186bbb7931c